admin/vagrant-acceptance-targets/centos-7/Vagrantfile

```diff
@@ -18,10 +18,6 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   rescue KeyError
   end
 
-  if Vagrant.has_plugin?("vagrant-cachier")
-    config.cache.scope = :box
-  end
-
```

Comment:
This is unrelated, but is useless, and was where the original change I was going to make was.